utils/publish.sh

```diff
@@ -32,19 +24,12 @@ fi
 
 if [[ "$tag" =~ -canary ]]; then
   echo "Publishing canary release"
-  npm_tag="--npm-tag canary"
+  dist_tag="--dist-tag canary"
```

Comment:
The lerna flag was renamed to `--dist-tag`, so this fixes a deprecation warning.